src/XrdCeph/XrdCephPosix.cc

```diff
@@ -684,6 +684,31 @@ int ceph_posix_open(XrdOucEnv* env, const char *pathname, int flags, mode_t mode
   if ((flags&O_ACCMODE) == O_RDONLY) {  // Access mode is READ
 
     if (fileExists) {
+      librados::bufferlist d_stripeUnit ;
```

Comment:
Just style: i think, it's better to have no space between variable name and semicolon (that seems to be the case for other variable declarations).